packages/components/fields/async-creatable-select-field/package.json

```diff
@@ -22,6 +22,7 @@
     "@commercetools-uikit/design-system": "10.18.4",
     "@commercetools-uikit/field-errors": "10.22.1",
     "@commercetools-uikit/field-label": "10.22.1",
+    "@commercetools-uikit/icons": "10.22.0",
```

Comment:
Since the `icons` are only used by the Storybook example (correct me if I'm wrong), I think we can remove them of the dependencies of the package and import locally (with relative path) in the story.

Reply:
will do that